yt/chyt/controller/internal/strawberry/oplet.go

```diff
@@ -984,6 +1101,14 @@ func (oplet *Oplet) Pass(ctx context.Context, checkOpLiveness bool) error {
 		return err
 	}
 
+	if oplet.strawberrySpeclet.ResumeMarker != oplet.persistentState.ResumeMarker {
+		// Maybe put into persistent state, so resuming will be not interrupted by strawberry restart?
+		oplet.pendingResume = true
+		oplet.persistentState.ResumeMarker = oplet.strawberrySpeclet.ResumeMarker
+		oplet.persistentState.YTOpSpeclet = oplet.specletYson
```

Comment:
YTOpSpeclet/YTOpSpecletRevision relate to the speclet/revision the operations was started with. It should be changed only during the operation start.

And I don't understand the `pendingResume` flag purpose. We can compare persistentState.ResumeMarker vs speclet.ResumeMarker in `needsResume` and update persistentState.ResumeMarker in `resumeOp` - this looks more reasonable to me.